src/DependencyInjection/Configuration.php

```diff
@@ -159,10 +159,6 @@ public function getConfigTreeBuilder()
                             ->defaultFalse()
                             ->info('Enable locking when editing an object, if the corresponding object manager supports it.')
                         ->end()
-                        ->booleanNode('enable_jms_di_extra_autoregistration') // NEXT_MAJOR: remove this option
```

Comment:
What if someone has this option in his config ? 

The class couldn’t be used but theorically someone could have this in his config and the update will throw an error. Isn’t it ?

Reply:
I have checked, no it will not thow error. But I may be wrong. 
"Works on my machine" ©